core/src/main/scala/scalafix/syntax/package.scala

```diff
@@ -25,6 +37,51 @@ package object syntax {
       if (config.hasPath(key)) config.getBoolean(key)
       else els
   }
+  implicit class XtensionCompleted[T](completed: Completed[T]) {
+    // TODO(olafur) contribute these upstream
+    def toOption: Option[T] = completed match {
+      case Completed.Success(e) => Some(e)
+      case _ => None
+    }
+    def toEither: Either[Exception, T] = completed match {
+      case Completed.Success(e) => Right(e)
+      case Completed.Error(e) => Left(e)
+    }
+    def toTry: Try[T] = Try(completed.get)
+  }
+  implicit class XtensionSymbol(symbol: Symbol) {
+    /** Returns simplified version of this Symbol.
+      *
+      * - No Symbol.Multi
```

Comment:
Then you need to write a custom rewrite. I made this normalized symbol to simplify the interface for authors writing basic rewrites, basically so it can be done via simple config file.